applications/nrf_desktop/README.rst

```diff
@@ -8,15 +8,25 @@ Nordic Semiconductor HID reference design.
 Overview
 ********
 
-This application is a reference design of a HID device.
-
 The code is currently work-in-progress and as such is not fully functional, verified, or
 supported for product development.
 
+Welcome to the firmware documentation for nRF52 Desktop, a reference design of
+a HID device.
+
+Depending on a configuration this application can function as mouse,
+gaming mouse or keyboard connected to host via BLE and/or USB.
+
+Project supports common input HW interfaces i.e. motion sensors, rotation
+sensors and buttons scanning module.
+Firmware can be configured at run-time using dedicated configuration channel
+realized with HID feature report. The same channel is used to pass DFU packets.
+
+
 Requirements
 ************
 
-Supported boards:
+The project contains configuration files for following boards:
```

Comment:
Also, how do I know which file is for which board? Where can I find these files?

Reply:
I plan to give more detail in configuration guide and would prefer to limit the number of information given in this section to supported boards.